.github/workflows/build.yml

```diff
@@ -30,6 +30,7 @@ jobs:
         pip install -U pytest
         pip install -r requirements.txt
         pip install -r bin/requirements.txt
+        echo "y" | pip uninstall epispot
```

Comment:
Did you install epispot?

Reply:
I see, but then why is `epispot` in the `bin/requirements.txt` file in the first place?

Reply:
Yes, via `bin/requirements.txt`
Epispot is a requirement for the CLI. In order to stop the epispot package from PyPI from interfering with the source package, we must uninstall epispot. This is the first major fix of this PR (see description)

Reply:
Why not just move fire to the base requirements.txt file and then delete the one in bin? It makes it easier for people cloning the git repo and not installing the package from PyPI. Also, you don't include the package itself in the package's requirements. The file serves as a way to show all _external_ dependencies which the user installs.

Reply:
Because it is a requirement for the CLI. The CLI is treated as a separate application from the main package.

Reply:
Nope, it is part of the package and is shipped with the package, meaning that the user has the package already installed. Once again, the `requirements.txt` file serves as a file that users cloning the git repo need to install, meaning all the external dependencies.